frontend/src/core/codemirror/go-to-definition/commands.ts

```diff
@@ -44,14 +44,6 @@ export function goToVariableDefinition(
         return false;
       } // Stop traversal if found
 
-      // Skip function/lambda bodies entirely
-      if (
-        node.name === "LambdaExpression" ||
-        node.name === "FunctionDefinition"
-      ) {
-        return false;
-      }
-
```

Comment:
I added this block in #5415 while trying to fix go-to-definition for scoped variables, but I think the issue was resolved elsewhere. I haven’t been able to reproduce the original problem, so I suggest we revert it.